src/containers/ClassifierContainer.jsx

```diff
@@ -231,7 +320,8 @@ class ClassifierContainer extends React.Component {
               <span>Shortcuts</span>
             </button>
 
-            {(!(isAdmin && this.props.previousAnnotations && this.props.previousAnnotations.length > 0)) ? null : (
+            {/*  //TODO: Will, is this still necessary?
```

Comment:
I think we can remove this. It was needed a while back when we were removing some bugs but is no longer necessary.